Sources/System/MachPort.swift

```diff
@@ -0,0 +1,318 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+#if $MoveOnly && (os(macOS) || os(iOS) || os(watchOS) || os(tvOS))
+
+import Darwin.Mach
+
+protocol MachPortRight {}
+
+enum Mach {
+    @_moveOnly
+    struct Port<RightType:MachPortRight> {
```

Comment:
Style: All type spacing in this file should be `RightType: MachPortRight`, `var name: mach_port_name_t` etc.